### PR TITLE
NEXT: Tail Glow

### PR DESCRIPTION
**Pull request by DeathlyPlays**

### CODE REVIEW

File: mods/gennext/README.md [hunk 1]
```diff
@@ -391,6 +391,8 @@ Minor learnset changes:
 
 - Scolipede and Steelix get Coil
 
+- Raichu, Lumineon, Ampharos, Chinchou, Lanturn get Tail Glow
```
Comment (Zarel) [inline mods/gennext/README.md]: We never agreed on Raichu :[

Comment (Zarel) [reply mods/gennext/README.md]: And prevos like Chinchou aren't mentioned in the README.

Comment (DeathlyPlays) [reply mods/gennext/README.md]: Chinchou is mentioned lol

Comment (Marty-D) [reply mods/gennext/README.md]: He meant prevos aren't supposed to be mentioned in the README, but you did mention one, and now even more.

Comment (DeathlyPlays) [reply mods/gennext/README.md]: Ohhh my bad

**Pull request merged**